services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/TestExtensions.kt

```diff
@@ -22,7 +22,10 @@ inline fun <reified T : Any> String.asJsonResource(): T =
 
 /** Read the response body from a MockMvc result and deserialize from JSON. */
 inline fun <reified T> ResultActions.andReturnBody(): T =
-    OBJECT_MAPPER.readValue(andReturn().response.getContentAsString(StandardCharsets.UTF_8))
+    OBJECT_MAPPER.readValue(andReturnContent())
+
+fun ResultActions.andReturnContent(): String =
```

Comment:
While rebasing, I added this method to make getting the content string in the correct charset easier.